### ISSUE TITLE
The library is difficult to use from outside it's main folder.

### ISSUE THREAD

**Issue by shailesh1729** [andrewssobral/lrslibrary#2]
I added the main folder on MATLAB path and tried to use process_matrix function from outside. In turn it internally calls commands like:``addpath('algorithms/lrr/ALM');`` This doesn't work if I am operating from a folder different from the lrslibrary folder. Consequently, I cannot run the algorithms. I bypassed by manually adding paths of the specific algorithms I wanted on MATLAB Path.  

Further, adding the lrslibrary folder into MATLAB path is also not very clean. It contains too many scripts whose names may clash with similarly named scripts in other folders. 

Basically an installation script is needed.

**Comment by andrewssobral** [comment]
Hi @shailesh1729, thank you for your suggestion, I'll check this.

**Comment by shailesh1729** [comment]
For the add path thing, you can probably do it using

curfile = which(mfilename)

and then use fileparts and fullfile commands to create absolute paths. This
will ensure that paths are picked up properly.

**Comment by andrewssobral** [comment]
Hi @shailesh1729, please check the last version of LRSLibrary. Best regards.

**Comment by shailesh1729** [comment]
Thanks Andrews,
I was travelling. Will check it out as soon as possible.

With regards,
- Shailesh

**Comment by shailesh1729** [comment]
Yeah it works after running the setup program. Thanks a lot.
